lib/msal-common/src/request/RequestValidator.ts

```diff
@@ -28,14 +28,7 @@ export class RequestValidator {
      * @param prompt
      */
     static validatePrompt(prompt: string) : void {
-        if (
-            [
-                PromptValue.LOGIN,
-                PromptValue.SELECT_ACCOUNT,
-                PromptValue.CONSENT,
-                PromptValue.NONE
-            ].indexOf(prompt) < 0
-        ) {
+        if (Object.values(PromptValue).indexOf(prompt) < 0) {
```

Comment:
I thought `Object.values` is available only in `es6` and hence we didn't use it before. Not supported in IE - [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Object/values). Is this okay?

Reply:
Yeah, `Object.values` should be avoided for now.

Reply:
Good point. Updated.